aria/orchestrator/workflows/builtin/utils.py

```diff
@@ -51,14 +56,14 @@ def create_node_task_dependencies(graph, tasks_and_nodes, reverse=False):
 
     def get_task(node_id):
```

Comment:
Since you will be fixing the builtins in ARIA-120, I want to suggest that perhaps much of these needs to be reworked, and it is better to be reworked there.